index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+  <title>portfolio</title>
+</head>
+
+<body>
+  <header>
+    <nav class="nav-bar">
+      <h2>tamana balkhi</h2>

```

Comment:
[OPTIONAL] Hey @tamana-Balkhi, it would be really nice to make this a link tag that redirects users back to the homepage. This is for three reasons: convention, semantics, and user experience(UX).

You can check this link [nav-link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page)